src/components/AlbumList.jsx

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import { Album }  from './Album';
+
+export const AlbumList = ({ albums }) => {
+  return (
+    <div className="album-container">
+      {albums.map((album) => (
+        <Album key={album.id} album={album} />
+      ))}
+    </div>
+  );
+};
+
+AlbumList.propTypes = {
+  albums: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
```

Comment:
Again, prop types is nice to understand. But we haven't really gone thru it yet.